index.html

```diff
@@ -11,28 +11,45 @@
 <body>
     <script src="js/bootstrpa.min.js"></script>
         <div class="container">
-            Container Fluid
+            
+            <p>[Container Fluid]</p>
 
-            <div class="header">
-                <div class="col-md-12">Header</div>
+            <div class="header row">
+                <div class="col-md-12">
+                    <p>[Header]</p>  
+                </div>
             </div>
             
-            <div class="Nav">
-                <div class="col-md-12 nav">Nav</div> 
-            </div>
+            <div class="row">
+                <div class="col-md-12 nav">
+                    <p>[Nav]</p>
+                </div> 
+            </nav>
 
             <section class="row">
-                    <main class="col-md-8">[Content]</main>
-                    <aside class="col-sm-4">[Aside]</aside>
+                    <main class="col-md-8">
+                        <p>[Content]</p>
+                    </main>
+                    <aside class="col-sm-4">
+                        <p>[Aside]</p>
+                    </aside>
             </section>
         
 
 
         <section class="Row-2">
```

Comment:
this looks great. go ahead and add class `row` with the `row-2` class. just to be safe for structuring.